src/configure/configure.ts

```diff
@@ -455,27 +455,29 @@ class Orchestrator {
                 throw error;
             }
         }
-        else if (!!node.value && node.value.nodeType === 'cluster') {
-            this.inputs.subscriptionId = node.value.subscription.subscriptionId;
+        else if (!!node.cloudResource && node.cloudResource.nodeType === 'cluster') {
+            this.inputs.subscriptionId = node.cloudResource.subscription.subscriptionId;
+            this.context['subscriptionId'] = this.inputs.subscriptionId;
             this.inputs.azureSession = getSubscriptionSession(this.inputs.subscriptionId);
             this.azureResourceClient = new AzureResourceClient(this.inputs.azureSession.credentials, this.inputs.subscriptionId);
-            const cluster = await this.azureResourceClient.getResource(node.value.armId, '2019-08-01');
+            const cluster = await this.azureResourceClient.getResource(node.cloudResource.id, '2019-08-01');
             telemetryHelper.setTelemetry(TelemetryKeys.resourceType, cluster.type);
             telemetryHelper.setTelemetry(TelemetryKeys.resourceKind, cluster.kind);
```

Comment:
what does `kind` represent here?

Reply:
I don't think "kind" should exist for cluster type resource. Will verify, and remove.